src/components/pages/rules/rules.js

```diff
@@ -1,14 +1,32 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 import React, { Component } from 'react';
-
+import RulesGrid from './rulesGrid/rulesGrid';
+import { Btn, RefreshBar } from 'components/shared';
 import './rules.css';
 
 export class Rules extends Component {
+  constructor(props) {
+    super(props);
+
+    if (!this.props.rules.length) {
+      this.props.fetchRules();
+    }
+  }
+
+  changeDeviceGroup = () => {
+    const { changeDeviceGroup, deviceGroups }  = this.props;
+    changeDeviceGroup(deviceGroups[1].id);
```

Comment:
isFunc?  Shouldn't we check that for everything we expect to be a function?

Reply:
See comment above.